astropy/utils/masked/function_helpers.py

```diff
@@ -163,16 +163,6 @@
     np.einsum, np.einsum_path,
 }  # fmt: skip
 
-# Really should do these...
-if NUMPY_LT_2_0:
-    from numpy.lib import arraysetops
-else:
-    # Public set operations have been moved to the top-level namespace in numpy 2.0
-    # (numpy/numpy#24507), raising an AttributeError when accessed through np.lib.arraysetops.
-    from numpy.lib import _arraysetops_impl as arraysetops
-
-IGNORED_FUNCTIONS |= {getattr(np, setopsname) for setopsname in arraysetops.__all__}
```

Comment:
Do we want to get rid of this set?

Reply:
Yes, because they are now all supported - these are now added to `MASKED_SAFE_FUNCTIONS` at the end.

Though perhaps I should just add them on top... I can do that tomorrow.

Reply:
Actually, did it now, since I also needed to add support for new `np.unique_*` functions in numpy 2.0